lib/ansible/modules/cloud/amazon/elb_target.py

```diff
@@ -207,6 +240,8 @@ def main():
             target_group_name=dict(type='str'),
             target_id=dict(type='str', required=True),
             target_port=dict(type='int'),
+            target_status=dict(type='str'),
```

Comment:
Can you provide the set of statuses here as a choices list?

Reply:
Done!